packages/@uppy/tus/src/index.js

```diff
@@ -346,8 +409,24 @@ module.exports = class Tus extends Plugin {
 
         this.uppy.emit('upload-success', file, uploadResp)
         this.resetUploaderReferences(file.id)
+        queuedRequest.done()
```

Comment:
![image](https://user-images.githubusercontent.com/7578559/63731836-0451f480-c88b-11e9-93f0-d6828b155b9c.png)

I got this error here once, but I'm not sure why, and I don't seem to be able to reproduce it.